app/js/components/leaflet/LeafMap.jsx

```diff
@@ -95,9 +95,9 @@ class LeafMap extends React.Component {
         ref={this.onRef.bind(this)}
         crs={mcCRS}
         center={[0, 0]}
-        zoom={-6}
-        maxZoom={5}
-        minZoom={-6}
+        zoom={0}
+        maxZoom={0}
+        minZoom={0}
```

Comment:
you could try playing with maxNativeZoom/minNativeZoom https://leafletjs.com/reference.html#gridlayer-option which lets you zoom in/out further while using the same tiles zoom level

Reply:
Not even my project but with the tiles this small, you desperately need to have less images as the DOM is making so many requests to get the images which slows things down 

Mostly this was just a draft change to show what's live on vcmap.github.io since Mochas only uploaded z0 tiles